ets-demo/.gitignore

```diff
@@ -0,0 +1,16 @@
+# file types to ignore
+*.pyc
+*.pyd
+*.so
+*.enamlc
```

Comment:
i think we can finally ditch the `.enamlc` files from the gitignore.

Reply:
nit-picky : possibly even the `.so` files. i don't think there are any compiled modules in traitsui, are there?

Reply:
I don't think they are needed, but at the same time if they are here we also don't want to track them. These are probably just copied from everywhere (I copied from traitsui) without much curating so that they can be copied again to other places, again without much curating...

Reply:
They are removed now nonetheless!